docs/tutorials/notifications-custom.rst

```diff
@@ -268,6 +268,10 @@ Kinto should load the listeners without errors:
     Starting subprocess with file monitor
     2016-01-21 16:21:59,941 INFO  [kinto.core.initialization][MainThread] Setting up 'redis' listener
 
+.. note::
+
+   You will need to install `kinto-redis` to use this listener.
```

Comment:
* rSt requires double back ticks
* Wouldn't it be better to have this note on top ?